packages/synapse-react-client/src/components/FeaturedResearch/FeaturedResearch.tsx

```diff
@@ -0,0 +1,361 @@
+          <Link
+            href={research.values[linkColIndex] ?? ''}
+            target="_blank"
+            sx={{
+              color: 'grey.1000',
+              textDecoration: 'none',
+              '&:hover': {
+                textDecoration: 'none',
+              },
+            }}
+          >
+            {research.values[titleColIndex]}
+          </Link>
+        </Typography>
+        <Stack direction={'row'} useFlexGap gap={'16px'} alignItems={'center'}>
+          <Typography
+            variant="overline"
+            fontSize={'14px'}
+            sx={{
+              backgroundColor: 'grey.300',
+              borderRadius: '3px',
+              padding: '4px 8px',
+              lineHeight: 'initial',
+            }}
+          >
+            {parseTags(tagsColIndex, research)?.[0] || ''}
+          </Typography>
+          <Typography lineHeight={'normal'} color={'grey.600'}>
+            {research.values[publicationDateColIndex] &&
+              formatDate(
+                dayjs(Number(research.values[publicationDateColIndex])),
+                'MMMM, YYYY',
+              )}
+          </Typography>
+        </Stack>
+      </Stack>
+      <CardMedia
+        component="img"
+        image={url}
+        alt="Research Card Image"
+        sx={{
+          flexShrink: 0,
+          width: '140px',
+          height: '93.8px',
+          borderRadius: '6px',
+          objectFit: 'cover',
+          marginLeft: { xs: 'initial', sm: 'auto' },
+        }}
+      />
+    </Box>
+  )
+}
+
+const FeaturedResearchTopCard = ({
+  research,
+  entityId,
+  affiliationColIndex,
+  titleColIndex,
+  descriptionColIndex,
+  tagsColIndex,
+  linkColIndex,
+  imageColIndex,
+  isLoading,
+}: FeaturedResearchCardProps) => {
+  const fileId = research.values[imageColIndex] ?? ''
+  const url = useImageUrl(fileId || '', entityId)
+  if (isLoading) {
+    return (
+      <Skeleton
+        variant="rectangular"
+        width="100%"
+        sx={{
+          height: { xs: '250px', sm: '500px', md: '600px', lg: '800px' },
+        }}
+      />
+    )
+  }
+  return (
+    <div>
+      <CardMedia
+        component="img"
+        image={url}
+        alt="Top Research Card Image"
+        sx={{
+          objectFit: 'cover',
+          maxHeight: { xs: '300px', md: '600.3px' },
+          borderRadius: '10px',
+          marginBottom: '30px',
+        }}
+      />
+      <Stack
+        direction="row"
+        useFlexGap
+        gap={'10px'}
+        paddingBottom={'20px'}
+        alignItems={'start'}
+      >
+        <Typography
+          fontSize={'14px'}
+          lineHeight={'normal'}
+          sx={{
+            fontWeight: 700,
+            backgroundColor: 'primary.main',
+            color: '#FFFF',
+            padding: '3px 8px',
+            borderRadius: '3px',
+          }}
+        >
+          {parseTags(tagsColIndex, research)?.[0] || ''}
```

Comment:
Intentional to only show the first tag? Seems like we should show a chip for each one, but maybe this is intentional to keep the design simple

Reply:
Is it possible there are no tags? You might want to only render this `Typography` component if `parseTags(tagsColIndex, research)[0]` exists

Reply:
Also seems like the `?` is unnecessary since `parseTags` always returns an array